src/output/CMakeLists.txt

```diff
@@ -2,13 +2,13 @@ set(output_definitions "")
 set(output_sources dvs_to_udp.hpp dvs_to_udp.cpp dvs_to_file.hpp dvs_to_file.cpp )
 set(output_libraries aer aestream_file)
 
-if (USE_TORCH)
-    set(output_definitions ${output_definitions} WITH_TORCH)
-    set(output_sources ${output_sources} dvs_to_tensor.hpp dvs_to_tensor.cpp)
-    if (NOT WITH_PYTHON)
-      set(output_libraries ${output_libraries} ${TORCH_LIBRARIES})
-    endif()
-endif()
+# if (USE_TORCH)
```

Comment:
We should probably clear up the comments at somepoint since we could always git checkout to find old code.
Also maybe a comment at the top of each file giving a breif overview of its function would make the code base more readable

Reply:
I'm doing a quick modern C++ refresher course atm, once im comfortable with the modern features, I can help with this

Reply:
Good point! I removed the comment now. I like your idea about the comment on top of the files. I'll resolve the point about the cmake torch setting now, but keep the textual comment part in the back of your head, please :-)